specs/query/oslc-query.html

```diff
@@ -96,13 +96,23 @@
             licenseURI: "https://www.apache.org/licenses/LICENSE-2.0",
           },
         ],
-
-        // name of the TC
-        wg: "OASIS OSLC Lifecycle Integration Core (OSLC Core) TC",
-        wgShortName: "oslc-core",
-
-        // URI of the public TC page
-        wgURI: "https://www.oasis-open.org/committees/oslc-core/",
+        
+        // =====  Overall OSLC OP settings =====
+
+        wg: "OASIS OSLC OP",
```

Comment:
I think it should be "OASIS Open Services for Lifecycle Collaboration (OSLC) OP", just checked other published PSDs.